src/main/java/View/InMemorySong.java

```diff
@@ -17,7 +17,7 @@ public class InMemorySong implements SongDsView {
      * Empty constructor.
      */
     public InMemorySong(){
-
+        this.name = "";
```

Comment:
Remove?

Reply:
Necessary to avoid null pointer exception